src/engine/bufferscalers/rubberbandwrapper.cpp

```diff
@@ -11,9 +11,46 @@ using RubberBand::RubberBandStretcher;
 
 namespace {
 
-mixxx::audio::ChannelCount getChannelPerWorker() {
+/// The function is used to compute the best number of channel per RB task,
+/// depending of the number of channels and available worker. This allows
+/// hardware if will less than 8 core to adjust the task distribution in the
+/// most optimum way. The following table provide the expected number of channel
+/// per task when using multi threading globally.
+///
+///  | NbOfCore | Stereo | Stem |
+///  |----------|--------|------|
+///  | 1        | 2      | 8    |
+///  | 2        | 1      | 4    |
+///  | 3        | 1      | 2    |
+///  | 4        | 1      | 2    |
+///  | 5        | 1      | 2    |
+///  | 6        | 1      | 2    |
+///  | 7        | 1      | 2    |
+///  | 8        | 1      | 1    |
```

Comment:
We should always use two channels (stereo) threads, unless the user insists to use single channels which is not recommended.

Reply:
Yes, that's already the case. This table is only valid if the user has requested multi threading (and confirm the warning prompt)

Reply:
I think we should sharpen the terminology here. Instead of just "multi threading" we should call it "process stereo channels in seperated threads" or similar.

Reply:
Fair enough, I will rework the tooltip in the preference window